### PR TITLE
#169258617 config postgres database engine

### PR DESCRIPTION
**Pull request by NiyoEric**
#### What does this PR do?
This PR helps us to configure the Postgres database engine.

#### Description of Task to be completed?
**Expected**
- Download Postgres  software
- Create `.env` file and save the connection string
- Create local development DB 
- Create local testing DB

#### How should this be manually tested?
 - `git clone https://github.com/andela/team-odd-bn-backend`
- `cd team-odd-bn-backend`
- `npm install`
-  `git checkout config-db-pg-169258617`
- `npm run dev` to run in development database
- `npm test` to run in testing database 

#### Any background context you want to provide?
If have created a temporary file called `temp-server.js` to test the database engine. we can delete it anytime.

#### What are the relevant pivotal tracker stories?
[#169258618](https://www.pivotaltracker.com/story/show/169258618)
#### Screenshots (if appropriate)
Testing environment

test_db

Development environment
dev_db

#### Questions:
N/A

### CODE REVIEW

**Review by WillyBoris — COMMENTED** [review]
Well Done! However, you still have to remove console.log

**Review by victkarangwa — APPROVED** [review]
Great job @NiyoEric.
For the houndCI reviews, let's just wait for eslint to be merged for the houndCI to review code semantics and style violation in a good manner.

**Comment by NiyoEric** [comment]
> Well Done! However, you still have to remove console.log

@WillyBoris console removed

**Review by hezronkimutai — APPROVED** [review]

File: .env.example [hunk 1]
```diff
@@ -0,0 +1,2 @@
+DATABASE_URL_DEV=''
+DATABASE_URL_TEST=''
```
Comment (julietezekwe) [inline .env.example]: End of line

File: package.json [hunk 2]
```diff
@@ -4,7 +4,8 @@
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "dev": "export NODE_ENV=development && nodemon src/temp-server.js ",
+    "test": "export NODE_ENV=testing && nodemon src/temp-server.js "
```
Comment (julietezekwe) [inline package.json:9]: You need not export NODE_ENV on develop. Develop is picked up automatically

File: src/temp-server.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+            '${username}',
+            '${telephone}'
+          ) returning *;
+        `);
+        await connection.end();
+    } catch (err) {
+        console.log('---- Error -----', err);
+    }
+};
+
```
Comment (julietezekwe) [inline src/temp-server.js]: This is not meant to be in the server if I understand what you are trying to achieve. Let's sync on this after standup.

**Review by julietezekwe — CHANGES_REQUESTED** [review]
Nice job Eric. However, there are some things I pointed out, please attend to them.
Also, resolve the houndci issues and squash your commit.

File: package.json (hunk 2 above)
Comment (NiyoEric) [reply package.json:9]: @julietezekwe I know that Sequelize by default `NODE_ENV=development`. Now, It is working let me remove it.

File: src/temp-server.js (hunk 3 above)
Comment (NiyoEric) [reply src/temp-server.js]: @julietezekwe you are right no need this file. I have used it to confirm the database connection locally.

**Review by julietezekwe — COMMENTED** [review]
You still have three commits. please squash them into one

**Review by nziokaivy — APPROVED** [review]

**Review by julietezekwe — APPROVED** [review]

**Pull request merged**